### PR TITLE
Improve Staking Pros and Cons section

### PR DESCRIPTION
**Pull request by DrW3RK**

### CODE REVIEW

File: docs/learn/learn-staking.md [hunk 1]
```diff
@@ -503,24 +503,37 @@ For more on chilling, see the "[How to Chill][]" page on this wiki.
...
-Keep in mind that when the system's staking rate is lower than the ideal staking rate, the annual
-nominal return rate will be higher, encouraging more users to use their tokens for staking. On the
-contrary, when the system staking rate is higher than the ideal staking rate, the annual nominal
-return will be less, encouraging some users to withdraw.
+- Earn rewards for contributing to the network's security through staking.
+- Low barrier of entry through [Nomination Pools](learn-nomination-pools.md).
+- Can choose up-to
+  {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominations" defaultValue={16}/> :polkadot }}
+  {{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }}
+  which can help decentralize the network through the sophisticated
```
Comment (filippoweb3) [inline docs/learn/learn-staking.md]:
```suggestion
 {{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }} validators which can help to decentralize the network through the sophisticated
```

File: docs/learn/learn-staking.md [hunk 2]
```diff
@@ -503,24 +503,37 @@ For more on chilling, see the "[How to Chill][]" page on this wiki.
...
+The ideal staking rate is a dynamic value - as the number of active parachains influences the
+available liquidity that is available to secure the network.
+
+Any divergence from the ideal staking rate will result in the distribution of a proportion of the
+newly minted tokens through inflation to go to the treasury.Keep in mind that when the system's
+staking rate is lower than the ideal staking rate, the annual nominal return rate will be higher,
+encouraging more users to use their tokens for staking. On the contrary, when the system staking
+rate is higher than the ideal staking rate, the annual nominal return will be less, encouraging some
+users to withdraw. For in-depth understanding, check the
+[inflation](learn-staking-advanced.md#inflation) section on the Wiki.
```
Comment (filippoweb3) [inline docs/learn/learn-staking.md]:
```suggestion
Any divergence from the ideal staking rate will result in the distribution of a proportion of the
newly minted tokens through inflation to go to the treasury. Keep in mind that when the system's
staking rate is lower than the ideal staking rate, the annual nominal return rate will be higher,
encouraging more users to use their tokens for staking. On the contrary, when the system staking
rate is higher than the ideal staking rate, the annual nominal return will be less, encouraging some
users to withdraw. For in-depth understanding, check the
[inflation](learn-staking-advanced.md#inflation) section on the Wiki.
```

File: docs/learn/learn-staking.md [hunk 3]
```diff
@@ -503,24 +503,37 @@ For more on chilling, see the "[How to Chill][]" page on this wiki.
...
 
 ### Cons of Staking
 
-- Tokens will be locked for about 28 days on Polkadot after unbonding, and seven days on Kusama.
-- Punishment in case of validator found to be misbehaving (see [slashing](#slashing)).
-- You want to use the tokens for a parachain slot.
+- Tokens will be locked away for about 28 days on Polkadot after unbonding, and seven days on
```
Comment (filippoweb3) [inline docs/learn/learn-staking.md]:
```suggestion
- Tokens will be locked away for about {{ polkadot: 28 days on Polkadot :polkadot }}{{ kusama: 7 days on Kusama :kusama }} after unbonding, and seven days on
```

**Review by filippoweb3 — APPROVED** [review]
lgtm, some minor things ;) (cannot check them on the local rendering

**Comment by DrW3RK** [comment]
Renders well. Merging it

**Pull request merged**